airflow/utils/session.py

```diff
@@ -29,7 +29,9 @@ def create_session():
     session = settings.Session()
     try:
         yield session
-        session.commit()
+        # Only Commit is a new or a modified object exists in the session
```

Comment:
I believe the intended comment on this line is: 
`# Only Commit if a new or a modified object exists in the session`